homeassistant/components/tuya/models.py

```diff
@@ -79,7 +82,7 @@ def get_update_commands(
 class DPCodeTypeInformationWrapper[T: TypeInformation](DPCodeWrapper):
     """Base DPCode wrapper with Type Information."""
 
-    DPTYPE: DPType
+    _DPTYPE: type[T]
```

Comment:
Wouldn't it be possible to set `_DPTYPE` for all the classes?

```suggestion
    _DPTYPE: type[T] = T
```

Reply:
Good to know.